### PR TITLE
Fixes Spotless and runs spotless

### PR DESCRIPTION
**Pull request by supertopher**
zipped files with JSON extension made this task sad

## What
`./gradlew build` failed for me on `:spotless` 

## How
there was a zip file with a .json extension.  we ignore that file now
also ran spotless

[this is the one line change fixing spotless](https://github.com/airbytehq/airbyte/pull/13040#discussion_r877508193) the rest is formatting only

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -103,6 +103,7 @@ def createSpotlessTarget = { pattern ->
         'secrets',
         'charts', // Helm charts often have injected template strings that will fail general linting. Helm linting is done separately.
         'resources/seed/*_specs.yaml', // Do not remove - this is necessary to prevent diffs in our github workflows, as the file diff check runs between the Format step and the Build step, the latter of which generates the file.
+        'airbyte-integrations/connectors/source-amplitude/unit_tests/api_data/zipped.json', // Zipped file presents as non-UTF-8 making spotless sad
```
Comment (supertopher) [inline build.gradle:106]: dear reviewer this is the only line that isn't formatting only

Comment (git-phu) [reply build.gradle:106]: looks great
I wonder why we have such a file though lol

**Review by git-phu — APPROVED** [review]
seems legit

File: build.gradle (hunk 1 above)
Comment (supertopher) [reply build.gradle:106]: yeah I didn't blame it, I'll ask whomever is the author

**Comment by supertopher** [comment]
We have some failing tests for connectors.  I think these failures were pre-existing but I'm going to hold off committing to master until I've looked a little harder

**Review by davinchia — APPROVED** [review]
Painful. Thanks for cleaning this up.

**Pull request merged**